.github/workflows/dh-backend-ci.yml

```diff
@@ -37,18 +39,35 @@ on:
   pull_request:
     branches:
       - main
-    paths:
-      - 'build/infrastructure/dh/**'
-      - 'apps/dh/api-dh/**'
-      - '.github/workflows/dh-backend-ci.yml'
-      - '.github/actions/publish-release-contents/action.yml'
 
 jobs:
-  dotnet_solution_ci:
-    uses: Energinet-DataHub/.github/.github/workflows/dotnet-solution-ci.yml@v8
+  #
```

Comment:
Should we also run 'ci-base.yml' or is this handled by other workflows?

Reply:
They have their own stuff like license-check-ci.yml it would be hard for me to implement it without running it twice, and i can't remove the other since EO uses this also